### ISSUE TITLE
Issues with DocumentNamespace and Checksum validation

### ISSUE THREAD

**Issue by anthonyharrison** [spdx/tools-python#451]
Attempting to validate a SPDX tag value results in the following errors

```
Invalid DocumentNamespace value dns:softwareassuranceguardian.com, must contain a scheme (e.g. "https:") and should not contain the "#" delimiter, line:5
VerificationCode doesn't match verifcode form, line:16
```

The namespace is compatible with the RFC and the error message does not appear to be correct.

The issue with the PackageVerificationCode:is becuase the SHA1 checkum isn't in lowercase although it does seem strange that the PackageChecksum value is valid with the checkum being valid in uppercase.

Test data is

```
SPDXVersion: SPDX-2.2
DataLicense: CC0-1.0
SPDXID: SPDXRef-DOCUMENT
DocumentName: SAG-PM generated SBOM
DocumentNamespace: dns:softwareassuranceguardian.com
Creator: Organization: dns:reliableenergyanalytics.com
Creator: Tool: SAG-PM Version: 1.2
Created: 2022-11-26T18:45:28Z
PackageName: apache-tomcat-9.0.69.zip
PackageVersion: 9.0.69
SPDXID: SPDXRef-Package-fc4a1bf0-78a0-43ca-b4a9-78adfb42138c
PackageSupplier: Organization: Apache Foundation
PackageDownloadLocation: https://dlcdn.apache.org/tomcat/tomcat-9/v9.0.69/bin/apache-tomcat-9.0.69.zip/
FilesAnalyzed: true
PackageLicenseInfoFromFiles: NOASSERTION
PackageVerificationCode: CE9F343C4BA371746FD7EAD9B59031AE34D8AFC4
PackageLicenseConcluded: NOASSERTION
PackageLicenseDeclared: NOASSERTION
PackageCopyrightText: NOASSERTION
PackageChecksum: SHA1: CE9F343C4BA371746FD7EAD9B59031AE34D8AFC4
LicenseID: LicenseRef-Unlicense
LicenseName: Unlicense
```

**Comment by maxhbr** [comment]
Which version of the tools-python are you using?

**Comment by anthonyharrison** [comment]
The latest one.

**Comment by maxhbr** [comment]
Hey Anthony, we will have a look.

**regarding lowercase vs. uppercase:**
the spec states that it should be a `Single line of text with 160 bit binary represented as 40 lowercase hexadecimal digits`: https://spdx.github.io/spdx-spec/v2-draft/package-information/#79-package-verification-code-field

same for the checksums

**Comment by armintaenzertng** [comment]
fixed by #452 and #453
